Add missing dustcleaner libcv ldflags.

--- a/dustcleaner/src/src/Makefile.am
+++ b/dustcleaner/src/src/Makefile.am
@@ -30,5 +30,5 @@ INCLUDES =\
 	-I$(includedir)		\
 	`pkg-config --cflags opencv` 
 
-LDADD = $(GIMP_LIBS) 
+LDADD = $(GIMP_LIBS) `pkg-config --libs opencv`
 
--- a/dustcleaner/src/src/Makefile.in
+++ b/dustcleaner/src/src/Makefile.in
@@ -194,7 +194,7 @@ INCLUDES = \
 	`pkg-config --cflags opencv` 
 
 
-LDADD = $(GIMP_LIBS) 
+LDADD = $(GIMP_LIBS) `pkg-config --libs opencv`
 subdir = src
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
 mkinstalldirs = $(SHELL) $(top_srcdir)/mkinstalldirs
