Better menu entry for the Sprocket Hole script.

--- a/sprocket-hole/elsamuko-slide-with-sprockets.scm
+++ b/sprocket-hole/elsamuko-slide-with-sprockets.scm
@@ -326,5 +326,5 @@ Newest version can be downloaded from ht
                     SF-COLOR      _"Sprocket Hole Color" '(0 0 0)
                     SF-COLOR      _"Sprocket Hole Shade" '(237 156 0)
                     )
-(script-fu-menu-register "elsamuko-sprocketholes" _"<Image>/Filters/Generic")
+(script-fu-menu-register "Sprocket-Holes" _"<Image>/Filters/Decor")
 
